app/src/main/res/drawable/nav_item_color_state.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
   <item android:color="@color/aptoide_orange" android:state_checked="true"/>
-  <item android:color="@android:color/black"/>
+  <item android:color="#8A000000"/>
```

Comment:
extract to color, need to check if color exist in design guidelines. If not we need to add it or change it according to what the design team asks to.